Kickstarter-iOS/Views/Controllers/CheckoutNavigationController.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+final class CheckoutNavigationController: UINavigationController {
```

Comment:
We haven't really been using the term `Checkout` with the new work, I wonder if this should be `PledgeNavigationController`?

Reply:
Or alternatively `RewardsCollectionNavigationController`

Reply:
I think the idea was just that this is the navigation controller that will be used for the checkout flow and that it could potentially serve up other animators for different controllers.. not so?